app/services/combined_audio_derivative_creator.rb

```diff
@@ -37,14 +36,13 @@ def generate
     end
     output_files = {}
     # Create the two output files:
-    ['mp3', 'webm'].each do |format|
+    ['m4a'].each do |format|
       output_files[format] = output_file(format)
       ffmpeg_args = args_for_ffmpeg(output_files[format].path)
       cmd.run(*ffmpeg_args, binmode: true, only_output_on_error: true)
     end
```

Comment:
Is there a reason to keep this as array.each, when it's only one format?  Repalce with just `output_files['m4a']` etc?

Reply:
Getting read of that `each`. You're right!